packages/xo-web/src/common/xo/index.js

```diff
@@ -3796,3 +3798,29 @@ const _callGithubApi = async (endpoint = '') => {
 export const getMasterCommit = () => _callGithubApi('/commits/master')
 
 export const compareCommits = (base, head) => _callGithubApi(`/compare/${base}...${head}`)
+
+export const uuidToLink = text => {
+  const regex = /\b[0-9a-f]{8}\b-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-\b[0-9a-f]{12}\b/g
```

Comment:
What happens with opaque refs? Do we try to match the UUID inside? If so, I think it doesn't make sense to try to match them as they don't correspond to XAPI UUIDs.

Reply:
Opaque refs are matched, but after discussing with @julien-f and @MathieuRA, it seems there is actually no way to get related objects info, so they are wrapped by the copiable component, like other UUIDs that don't match any objects.
The copiable component is added only on the UUID part of the opaque ref, I can update the regex to enable copying also the `OpaqueRef:` part, if needed.